scripts/build.d/pyside6

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+set -e
+
+SRCSYNC=${SRCSYNC:-git}
+
+if [ "${SRCSYNC}" == "tarball" ] ; then
+
+  display -e "\${SRCSYNC} ${SRCSYNC} not supported by this package"
```

Comment:
I cannot find tarball download for pyside.  It seems that the project requires people to clone from their git.

Please correct me if I am wrong.